snowpack/src/commands/dev.ts

```diff
@@ -365,29 +365,48 @@ export async function startServer(
     }
   }
 
-  for (const runPlugin of config.plugins) {
-    if (runPlugin.run) {
-      logger.debug(`starting ${runPlugin.name} run() workers`);
-      runPlugin
+  let outputExts: string[] = [];
+
+  for (const plugin of config.plugins) {
+    if (plugin.run) {
+      logger.debug(`starting ${plugin.name} run() workers`);
+      plugin
         .run({
           isDev,
           // @ts-ignore: internal API only
           log: (msg, data) => {
             if (msg === 'CONSOLE_INFO') {
-              logger.info(data.msg, {name: runPlugin.name});
+              logger.info(data.msg, {name: plugin.name});
             } else {
-              messageBus.emit(msg, {...data, id: runPlugin.name});
+              messageBus.emit(msg, {...data, id: plugin.name});
             }
           },
         })
         .then(() => {
-          logger.info('Command completed.', {name: runPlugin.name});
+          logger.info('Command completed.', {name: plugin.name});
         })
         .catch((err) => {
-          logger.error(`Command exited with error code: ${err}`, {name: runPlugin.name});
+          logger.error(`Command exited with error code: ${err}`, {name: plugin.name});
           process.exit(1);
         });
     }
+    if (plugin.resolve) {
```

Comment:
mind moving this into a `getOutputExts` helper function? Or, better get, a `getOutputExtensionMatch()` that just returns the matcher function?